onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.h

```diff
@@ -239,9 +239,12 @@ namespace Dml
             GetCapability(const onnxruntime::GraphViewer& graph,
                 const std::vector<const onnxruntime::KernelRegistry*>& kernel_registries) const final;
 
-        onnxruntime::common::Status Sync()
+        // Not to be confused with IExecutionProvider::Sync() const.  The DML provider handles 
+        // synchronization when copying inputs and outputs, therefore doesn't override the 
```

Comment:
>synchronization when copying inputs and outputs [](start = 11, length = 47)

Is this true? I thought ORT did support binding inputs and outputs to device-specific memory. Isn't the IOBinding supposed to be the one who calls Sync to ensure the EP's work has completed?

Reply:
Yes, the EP internally synchronizes.  The Sync method being replaced was not overriding the base class method, which is const (and does nothing)